app/assets/stylesheets/elements/_inputs.scss

```diff
@@ -0,0 +1,5 @@
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
+  -webkit-appearance: none;
```

Comment:
This disables the spinner site-wide. Is that appropriate?

Not saying I can think of a good place where a spinner _would_ be appropriate.

Reply:
I couldn't find any other `<input type="number">`s in the project, so we're okay for now.